### PR TITLE
Fixed operationIds for non-CRUD operations in ExpressRouteCrossConnections

### PR DESCRIPTION
**Pull request by anton-evseev**
Mirrors ExpressRouteCircuit's non-CRUD operations

---

This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by salameer** [comment]
Hi Any reason to do the changes to the operation IDs?

**Comment by MikhailTryakhov** [comment]
@salameer  that is needed to save common style in express routes commands and make this code generatabe for PS. I wait for Vinaya to approve

**Comment by dsgouda** [comment]
@MikhailTryakhov please note that this is against the patterns for operationIds ARM recommends

**Comment by MikhailTryakhov** [comment]
@dsgouda  but such style for operation naming is already approved to top-level resource... Our idea was to save similar naming for this set of commands...

File: specification/network/resource-manager/Microsoft.Network/stable/2018-02-01/expressRouteCrossConnection.json
Comment (vinayada1) [inline specification/network/resource-manager/Microsoft.Network/stable/2018-02-01/expressRouteCrossConnection.json]: We missed this....could you rename express route circuit in the description to express route cross connection?

**Review by vinayada1 — APPROVED** [review]
Just a minor comment on the description,

**Review by dsgouda — APPROVED** [review]
LGTM pending @vinayada1 's comments

**Comment by number213** [comment]
Addressed @vinayada1's comment

**Pull request merged**
